main.tf

```diff
@@ -0,0 +1,12 @@
+
+resource "null_resource" "ansible_provisioner" {
+
+  provisioner "local-exec" {
+    command = "ansible-playbook ${var.playbook} -e host=${var.ip}"
```

Comment:
As we are not creating inventory file, we need to describe how to create inventory made of `host` variable.
Let's add this to README.md